docs/antrea-network-policy.md

```diff
@@ -1296,6 +1301,47 @@ Note: Antrea will use a reserved label key for internal processing `serviceAccou
 The reserved label looks like: `internal.antrea.io/service-account:[ServiceAccountName]`. Users should avoid using
 this label key in any entities no matter if a policy with `serviceAccount` is applied in the cluster.
 
+### ACNP appliedTo NodePort Service
+
+Antrea ClusterNetworkPolicy features a `service` field in `appliedTo` field to enable the ACNP could be enforced
+on the traffic from external client to a NodePort Service.
+
+`service` uses `namespace` and `name` to select the Service with a specific name under a specific namespace and
+only a NodePort Service can be referred by `service` field.
+
+`service` field cannot be used with any other fields and a policy or a rule can't be applied to NodePort Service
```

Comment:
Better to add "," before "and a policy"

Reply:
"both a NodePort Service and other entities".

Reply:
Done.